_docs/_latest/develop/branch-methodology.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Branch Methodology
+order: 003
+---
+
+#  {{ page.title }}
+
+## Branches
+
+There are several important main branches in Forseti repo to know about,
+in order to correctly inspect the right code, or create a Pull Request (PR).
+
+* `dev`: development branch for Forseti 1.0.
```

Comment:
I would make this document version agnostic. With that in mind, what do we use dev, master, and forsetisecurity.org for?

Additionally, when do we decide to create feature branches (criteria) and how will they be named, merged, and closed.

Reply:
Hmmm, referencing the specific version branches solves a real problem that users would look at our branches and not sure which branch to make a contribution.  So, I really recommend that we keep it.

As for specifically how to create the branches and how they are named, etc.... they seem very general git knowledge that's not specific to Forseti.  Do you think that we can assume developers would know them?  I'm happy to add them if you can confirm.  Thanks!

Reply:
Done.  I have updated the doc with your suggestion. I have added an version agnostic statement about the highest version branches, but it seems to me to still be good to list and describe what main branches we have.